src/rgw/driver/rados/cls_fifo_legacy.cc

```diff
@@ -547,8 +547,6 @@ void FIFO::_update_meta(const DoutPrefixProvider *dpp, const fifo::update& updat
   update_meta(&op, info.version, update);
   auto updater = std::make_unique<Updater>(dpp, this, c, update, version, pcanceled,
 					   tid);
-  auto r = ioctx.aio_operate(oid, Updater::call(std::move(updater)), &op);
```

Comment:
Why is this op and the assert below it being removed? This does not seem like clean up.